.github/workflows/swiftlint.yaml

```diff
@@ -20,10 +20,10 @@ jobs:
   swiftlint:
     runs-on: ubuntu-latest
     container:
-      image: ghcr.io/realm/swiftlint:5.5-0.48.0
+      image: ghcr.io/realm/swiftlint:latest
```

Comment:
This does not look right, it does revert all of the changes made in main branch. Same for the rest of the files.